pkg/certifier/attestation/attestation_vuln.go

```diff
@@ -0,0 +1,82 @@
+//
+// Copyright 2022 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package attestation_vuln
+
+import (
+	"time"
+
+	intoto "github.com/in-toto/in-toto-golang/in_toto"
+)
+
```

Comment:
would be helpful to also maybe add a paragraph to document the behavior or the parser, and how vuln attestations and vuln nodes are created within the graph. Just a prose will do, not required to be overly formal.